smaily-for-wp.php

```diff
@@ -19,7 +19,7 @@
 // Exit if accessed directly.
 if ( ! defined( 'ABSPATH' ) ) exit;
 
-define( 'SS_PLUGIN_VERSION', '1.2.3' );
+define( 'SNSP_PLUGIN_VERSION', '2.0.0' );
```

Comment:
Use something like `SMLY4WP` instead of `SNSP`. Latter can create unwanted conflicts with other plugins, or at least the probability of that happening is greater.